bin/npack

```diff
@@ -113,7 +117,7 @@ program.command('install <source>')
 			function(err, config, password) {
 				var installOptions = _(options)
 					.chain()
-					.pick('force', 'log', 'use', 'dir')
+					.pick('force', 'log', 'use', 'dir', 'useLockfile')
```

Comment:
Is it better to call this option `useShrinkwrapFile`? because package-lock.json cann't be published

Reply:
If so, then it should be renamed everywhere (error text, test labels, etc)